pkgs/development/tools/tabnine/default.nix

```
@@ -0,0 +1,33 @@
+{ stdenv, lib, fetchurl }:
+
+let
+  target =
+    if stdenv.hostPlatform.system == "x86_64-darwin" then
+      "x86_64-apple-darwin"
+    else if stdenv.hostPlatform.system == "x86_64-linux" then
+      "x86_64-unknown-linux-musl"
+    else throw "Not supported on ${stdenv.hostPlatform.system}";
+in stdenv.mkDerivation rec {
+  pname = "tabnine";
+  version = "3.1.1";
+
+  src = fetchurl {
+    url = "https://update.tabnine.com/${version}/${target}/TabNine";
+    sha256 = "w+Ufy4pICfQmseKCeohEQIP0VD6YrkYTEn41HX40Zlw=";
+  };
+
+  phases = [ "installPhase" ];
```

specifying individual phases is deprecated
```suggestion
  dontBuild = true;
```